modules/ebi-metagenomics/crisprcasfinder/meta.yml

```diff
@@ -0,0 +1,55 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "crisprcasfinder"
+description: crisprcasfinder
+keywords:
+  - sort
```

Comment:
replace with another keyword that actually does something here

Reply:
changed